tests/pages/firefox/new.py

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage, FirefoxBasePageRegion
+
+
+class FirefoxNewPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/new/{params}'
+
+    _download_button_locator = (By.CSS_SELECTOR, '#features-download .download-button .download-link')
+    _thank_you_message_locator = (By.ID, 'scene2-main')
+
+    @property
+    def download_buttons(self):
+        return [DownloadButton(self, root=el) for el in
+                self.find_elements(self._download_button_locator)]
+
+    @property
+    def platform_download_button(self):
+        return next(s for s in self.download_buttons if s.is_displayed)
+
+    @property
+    def is_download_button_displayed(self):
+        return self.platform_download_button.is_displayed
```

Comment:
I think we can get away without a page region for the download button. As far as I can tell we only really need to click the download button (which implicitly checks that it's visible). We could therefore do something like:

```python
def download_firefox(self):
    els = [el for el in self.find_elements(self._download_button_locator) if el.is_displayed]
    assert len(els) == 1, 'Expected one download button to be displayed'
    els[0].click()
```